src/getErrors.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+const ERRORS = {
+  INVALID_CASE: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+  NO_TEXT: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  NO_CASE: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+};
+
+const allowedCases = [
+  'SNAKE',
+  'KEBAB',
+  'CAMEL',
+  'PASCAL',
+  'UPPER',
+];
```

Comment:
I bet you will be asked to move these constants into a separate file).
You may consider using the allowedCases array in the INVALID_CASE error text.

Reply:
Not fixed